base-apache/Dockerfile.php7

```diff
@@ -81,10 +81,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   && apt-get clean -y \
   && rm -rf /var/lib/apt/lists/*
 
-# Install wkhtmltopdf
-RUN wget -O /wkhtml.deb https://downloads.wkhtmltopdf.org/0.12/0.12.5/wkhtmltox_0.12.5-1.${OS_VERSION}_amd64.deb \
-    && dpkg -i --force-depends /wkhtml.deb \
-    && rm /wkhtml.deb
+# Dependencies for Puppeteer headless chrome browser
+RUN apt-get update && apt-get install -y gconf-service libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 \
+    libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 \
+    libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 \
+    libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget && apt-get autoremove -y && apt-get clean -y
```

Comment:
Should use `--no-install-recommends` for `apt install` in docker files. Otherwise it tends to install a ton of extra dependencies that are not needed.

It is also best practice in dockerfiles to put each application on a single line - just becuase it makes it easier to read.

Reply:
No worries, sorted those out.

I noticed we've got `wget` in both lists, but I figured it would be easier to keep it in both for now, just so we can keep the dependencies atomic.